test/templates/action-bar/index.njk

```diff
@@ -12,87 +10,32 @@
     .fd-action-bar {
         margin: 0 20px;
     }
```

Comment:
Can we use the placeholder buttons already in layout-shared: https://github.com/SAP/fundamental-styles/blob/master/test/resources/layout-shared.css#L6

Reply:
done!